config/csi/kustomization.yaml

```diff
@@ -2,44 +2,5 @@
 namespace: ibm-spectrum-scale-csi
 
 resources:
-- github.com/IBM/ibm-spectrum-scale-csi.git?ref=v2.5.1
+- github.com/IBM/ibm-spectrum-scale-csi.git//operator/config/overlays/cnsa?ref=v2.6.0
```

Comment:
not familiar at all with this...  but assuming it's correct?

Reply:
Yes, management of the CSI images now resides in their repository.